usmqe_tests/api/gluster/test_crud_gluster_volume.py

```diff
@@ -0,0 +1,301 @@
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                job should finish.
+                """
+    volume_conf_2rep["Volume.volname"] = VOLUME_NAME
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+
+    job_id = api.create_volume(
+        cluster_reuse["cluster_id"],
+        volume_conf_2rep)["job_id"]
+    if api.wait_for_job_status(job_id) == "finished":
+        """@pylatest api/gluster.create_volume
+            API-gluster: create_volume
+            ******************************
+
+            .. test_metadata:: author fbalak@redhat.com
+
+            Description
+            ===========
+
+            Check if there is created volume on gluster nodes via CLI.
+
+            .. test_step:: 2
+
+                Connect to gluster node machine via ssh and run
+                ``gluster volume info command``
+
+            .. test_result:: 2
+
+                There should be listed gluster volume named ``Vol_test``.
+
+                """
+        storage = gluster.GlusterCommon()
+        storage.find_volume_name(VOLUME_NAME)
+
+        volume = gluster.GlusterVolume(VOLUME_NAME)
+        volume_id = volume.get_volume_id()
+        storage_volume_attributes = {
+                "name": volume.name,
+                "id": volume.id,
+                "status": volume.status,
+                "stripe_count": volume.stripe_count,
+                "replica_count": volume.replica_count,
+                "brick_count": volume.brick_count,
+                "snapshot_count": volume.snap_count
+            }
+        volumes = api.get_volume_list(cluster_reuse["cluster_id"])
+        volume_tendrl = [volume_t[volume_id] for volume_t in volumes
+                         if volume_id in volume_t]
+        pytest.check(
+            len(volume_tendrl) == 1,
+            """There should be only one volume
+            with id == {}""".format(volume_id))
+        if len(volume_tendrl) == 1:
+            volume_tendrl = volume_tendrl[0]
+            tendrl_volume_attributes = {
+                    "name": volume_tendrl["name"],
+                    "id": volume_tendrl["vol_id"],
+                    "status": volume_tendrl["status"],
+                    "stripe_count": volume_tendrl["stripe_count"],
+                    "replica_count": volume_tendrl["replica_count"],
+                    "brick_count": volume_tendrl["brick_count"],
+                    "snapshot_count": volume_tendrl["snap_count"]
+                }
+            pytest.check(
+                tendrl_volume_attributes == storage_volume_attributes,
+                """Storage volume attributes: {}
+                Tendrl volume attributes: {}
+                These should be the same.""".format(
+                    tendrl_volume_attributes, storage_volume_attributes))
```

Comment:
Why to create dictionary for tendrl volume attributes and gluster volume attributes and then compare those instead of compare parameters individually? Check that tendrl name == gluster name, tendrl id == gluster id etc. That will take almost the same amount of code and it will be more readable as the difference will be found more quickly.